examples/client-compilation.jade

```diff
@@ -0,0 +1,140 @@
+//
+  This example shows how to compile and render Jade templates in the browser.
+
+  Note that although it is supported, we do not *recommend* compiling in the
+  browser; rather, you should pre-compile the template using compileClient
+  or compileFileClient.
```

Comment:
It might be better to actually put this text as a paragraph on the HTML page as so people see it if they open the sample in their browser.